api/handlers_test.go

```diff
@@ -253,174 +259,207 @@ func TestFetchTransactions(t *testing.T) {
+		// 		loadTransactionFromFile("test_resources/app_call_1.response"),
+		// 	},
+		// 	created: 10,
+		// },
+		// {
+		// 	name: "Application Call (2)",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_call_2.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_call_2.response"),
+		// 	},
+		// 	created: 10,
+		// },
+		// {
+		// 	name: "Application Call (3)",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_call_3.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_call_3.response"),
+		// 	},
+		// 	created: 10,
+		// },
+		// {
+		// 	name: "Application Clear",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_clear.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_clear.response"),
+		// 	},
+		// },
+		// {
+		// 	name: "Application Close",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_close.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_close.response"),
+		// 	},
+		// },
+		// {
+		// 	name: "Application Update",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_update.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_update.response"),
+		// 	},
+		// },
+		// {
+		// 	name: "Application Delete",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_delete.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_delete.response"),
+		// 	},
+		// },
+		// {
+		// 	name: "Application Non ASCII Key",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_nonascii.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_nonascii.response"),
+		// 	},
+		// },
+		// {
+		// 	name: "Application Optin",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_optin.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_optin.response"),
+		// 	},
+		// },
+		// {
+		// 	name: "Application With Foreign App",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_foreign.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_foreign.response"),
+		// 	},
+		// },
+		// {
+		// 	name: "Application With Foreign Assets",
+		// 	txnBytes: [][]byte{
+		// 		loadResourceFileOrPanic("test_resources/app_foreign_assets.txn"),
+		// 	},
+		// 	response: []generated.Transaction{
+		// 		loadTransactionFromFile("test_resources/app_foreign_assets.response"),
+		// 	},
+		// },
```

Comment:
Don't forget to re-enable the old tests before merging

Reply:
My bad, disabled when was fixing ids. Restored now.